vllm/model_executor/layers/quantization/__init__.py

```diff
@@ -78,6 +80,10 @@ def get_quantization_config(quantization: str) -> Type[QuantizationConfig]:
         raise ValueError(f"Invalid quantization method: {quantization}")
 
     # lazy import to avoid triggering `torch.compile` too early
+    from vllm_hpu_extension.awq_hpu import AWQHPUConfig
+
+    from vllm_hpu_extension.gptq_hpu import GPTQHPUConfig
```

Comment:
Shouldn't this only be imported when we are hpu device?

Reply:
@maktukmak , please help to fix